### ISSUE TITLE
error when running "php artisan migrate"

### ISSUE THREAD

**Issue by pphead332** [Flofy-Dev/rainway-source#34]
PHP Warning:  require(C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main\artisan on line 18

Warning: require(C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main/vendor/autoload.php): Failed to open stream: No such file or directory in C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main\artisan on line 18
PHP Fatal error:  Uncaught Error: Failed opening required 'C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main\artisan:18
Stack trace:
#0 {main}
  thrown in C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main\artisan on line 18

Fatal error: Uncaught Error: Failed opening required 'C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main/vendor/autoload.php' (include_path='C:\xampp\php\PEAR') in C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main\artisan:18
Stack trace:
#0 {main}
  thrown in C:\Users\cakin\Downloads\rainway-source-main\rainway-source-main\artisan on line 18

**Comment by squishdotcafe** [comment]
run composer update or composer install first

that will generate the autoload files

**Comment by Flofy-Dev** [comment]
yeah sorry for no solution lol this source is just an archive, better late than never ahh response from me. thanks @squishdotcafe
